content/10.diagnostics.md

```diff
@@ -162,6 +199,20 @@ Both techniques were equally sensitive in detecting SARS-CoV-2.
 Lateral flow strips showed a 100% correlation to the high-throughput DETECTR assay.
 Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.
 
+#### Protein tests
+LFT-based tests are beneficial because they can detect current infection within 30 minutes and because they can be performed without specialized equipment at low cost. 
+LFTs rely on detection of viral protein with an antibody. 
+Often this is done with an antibody sandwich format, where one antibody containing a dye is binds at one site on the antigen, and an immobilized antibody on the strip binds at another site, allowing accumulation of the dye and the characteristic formation of a positive test line on the strip [@URL:https://doi.org/10.1007/s00216-010-3661-4].
+The applications of LFTs are broad: they are routinely used for home pregnancy tests, disease detection, and even drugs of abuse detection in urine [@DOI:10.1007/s00216-008-2287-2].
+Current infection detection by LFTs is needed to enable the scale and speed of testing required to stop the spread of SARS-CoV-2.
+Lateral flow tests were made available freely to citizens in the United Kingdom at least until December 2021 [@DOI:10.1136/bmj.n1760].
```

Comment:
```suggestion
LFTs were made available freely to citizens in the United Kingdom at least until December 2021 [@DOI:10.1136/bmj.n1760].
```

Reply:
I think multiple regions now or at least at a very low price in Europe

Reply:
Trying to find something citable that shows LFTs are free right now in the UK - is it possible to cite this website where they are ordered? https://www.nhs.uk/conditions/coronavirus-covid-19/testing/regular-rapid-coronavirus-tests-if-you-do-not-have-symptoms/

Reply:
The citation that is in there talks about how they are free last summer and that they might end them being free at the end of august.